handlers/handler-flowcontrol-qps/src/main/java/org/apache/servicecomb/qps/QpsControllerManager.java

```diff
@@ -151,7 +151,17 @@ private AbstractQpsStrategy searchQpsController(String qualifiedNameKey) {
       return qpsStrategy;
     }
 
-    return null;
+    return new AbstractQpsStrategy() {
```

Comment:
This is not correct. Look at where searchQpsController is called. 

Maybe you can look at the code in `create`, move logic in `create` of search gloableQpsStrategy to searchQpsController  can solve the problem .